schemas/1.21.0

```diff
@@ -50,6 +50,33 @@ versions:
         # https://github.com/open-telemetry/semantic-conventions/pull/53
         - rename_metrics:
             process.runtime.jvm.cpu.utilization: process.runtime.jvm.cpu.recent_utilization
+        # https://github.com/open-telemetry/opentelemetry-specification/pull/3464
+        - rename_attributes:
+            attribute_map:
+              memory: memory.type
```

Comment:
```suggestion
              type: memory.type
```
Isn't the attribute called `type` today?

Reply:
it's `from:to` - and it's meant to be renamed to `memory.type`

Reply:
you're right - fixed